### ISSUE TITLE
Database Schema Changes

### ISSUE THREAD

**Issue by bluek1te** [bluek1te/feed-txstate#16]
We need these in our database as a minimum to test:

_These are going to be the entries that are filled out by runners as they pick up food_
rescue_entry 
- rescue_id
- location_id (store_number)
- date
- time_pick_up
- time_drop_off
- food_name
- temp_pick_up (making meat + produce the same for this field and the next)
- temp_drop_off
- weight
- last_updated_by_user_id (last_updated_by)
- updated_at

location
- location_id 
- location_name (location)
- address

_These are going to help determine whether the food being picked up needs to be treated as meat or produce_
food_name_food_type
- food_name
- food_type

_This table used to relate users with locations and days to generate a route_
location_user_day
- location_name
- user
- day

user
- user_id
- user_name

We're just going to focus on rescues right now. Everyone, let me know if I'm missing anything or if you think these tables should be edited.

**Comment by BenjaminWalkerBond** [comment]
Can you explain why we are merging the temperatures variables into one? The rest makes a lot of sense.

**Comment by bluek1te** [comment]
Because if being a meat or a vegetable or a canned item is going to be mutually exclusive, we can make decision paths on what to do with the temperature based on the food type

**Comment by BenjaminWalkerBond** [comment]
That makes sense. This will affect the design of the review page and the temperature input page in the rescue route though. Ill make some quick changes to those, (Simplification of the temperature to one variable will actually help me resolve one of the bugs I haven't been able to solve).

I think I will make the necessary design changes to the database now before focusing on further connecting the database to the front end.

**Comment by BenjaminWalkerBond** [comment]
Also, might I suggest a field to keep track of cart weights? Either as its own table or as part of another.

**Comment by bluek1te** [comment]
Yeah, it's just cloudy to me of where cart weights come in during the rescue process. Because the cart is made of a combination of rescues, should we put it in with the rescue record?? Or should we make another table?

**Comment by BenjaminWalkerBond** [comment]
I'm tempted to make them their own table but I can't remember whether or not they use the carts weights on both rescue and distribution or not. I believe they use carts in both the rescue and distribution process so I will make a separate table for now.
